### ISSUE TITLE
pkgs.skaffold hash mismatch

### ISSUE THREAD

**Issue by fabifrank** [NixOS/nixpkgs#150388]
### Describe the bug
skaffold can no longer be installed due to hash mismatch:

```
specified: sha256-+VnZoZhBooLue+82zOngij8PpH9/wtZkYiyfklHSpsQ=
got:    sha256-8Ye2eR9eB7oyYOo46OraOxfLOG/XphWJkk+xPzDthPU=
```

### Steps To Reproduce
1. create `default.nix` with:
```
{ pkgs ? import <nixpkgs> {} }:
pkgs.mkShell {
        buildInputs = [
                pkgs.skaffold
        ];

}
```

2. run `nix-shell --pure`

### Expected behavior
nix-shell will install deps correctly and open a shell.

### Additional context
I believe it's because skaffold released 1.35.1 2 days ago and the hash in nix is 6 days ago.

### Notify maintainers
@r-ryantm tagging you as you've been the last to touch the file https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/skaffold/default.nix.

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `XXX`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.4`
 - channels(root): `"nixpkgs-22.05pre337269.33582b3ee1d"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixpkgs`

```

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

**Comment by fabifrank** [comment]
Apart from the bug: what would be the best way to circumvent issues like that, in example to rollout a local fix for it until a fix is merged in the official store?

**Comment by FliegendeWurst** [comment]
This PR should fix it: #150387

**Comment by ryantm** [comment]
> what would be the best way to circumvent issues like that

@fabifrank See https://nixos.wiki/wiki/Overlays "Overriding a version"
